src/utils/libraryChecker.ts

```diff
@@ -0,0 +1,23 @@
+import * as child_process from 'child_process';
+import { promisify } from 'util';
+
+const exec = promisify(child_process.exec);
+
+export async function libraryChecker(
```

Comment:
Can this be refactored to return the executable path by executing `which <name-of-executable>` and rename the function to `getExecutablePath`

Reply:
Yes sure.

Reply:
This have been implemented @ganeshrn. You can resolve this, if satisfactory.